catalog.html

```diff
@@ -196,23 +199,22 @@ <h3>Sedona  </h3>
             </ul>
         </section>
 
-        <section>
-            <img src="https://via.placeholder.com/1400x416" width="1400" height="416" alt=""> 
-            
+        <section><!--Карта+контакты+КартинкаМаркер-->
+            <img src="img/map1.jpg" width="1400" height="416" alt=""> 
```

Comment:
```suggestion
            <img src="img/map1.png" width="1400" height="416" alt=""> 
```
А вот для изображения карты (как и для фото текста, чертежа) лучше всего подходит формат PNG, так как в нём используется алгоритм сжатия без потери качества, чего не скажешь про JPEG/